src/stim/circuit/gate_data.test.cc

```diff
@@ -67,7 +68,8 @@ TEST(gate_data, hash_matches_storage_location) {
     }
 }
 
-std::pair<std::vector<PauliString>, std::vector<PauliString>> circuit_output_eq_val(const Circuit &circuit) {
+std::pair<std::vector<PauliString<MAX_BITWORD_WIDTH>>, std::vector<PauliString<MAX_BITWORD_WIDTH>>>
+circuit_output_eq_val(const Circuit &circuit) {
```

Comment:
The dependency on the `TableauSimulator` means that tests using this function cannot be tested for all SIMD widths until the `TableauSimulator` is templatised. The only test this impacts here is `decompositions_are_correct`.